### ISSUE TITLE
Frequent crashes

### ISSUE THREAD

**Issue by mjkalyan** [drobilla/patchage#2]
I'm getting frequent crashes, especially when I change the song I'm listening to via MPD (it'll happen nearly every time). I use PipeWire as a JACK replacement.

Here's the error output:
```
** (patchage:29114): ERROR **: 16:04:59.411:
unhandled exception (type std::exception) in signal handler:
what: argument not found
```

I don't have any insight otherwise as to what causes this.

**Comment by drobilla** [comment]
Hm, I can't reproduce this with other clients (I'm not using MPD right now and can't be bothered to set it up), although it is related to logging (that exception comes from `fmt`) and I just noticed I had to change something to get it to build with the latest release.

Maybe the formerly dynamic error became static, does 632dd70 fix it by any chance?

**Comment by mjkalyan** [comment]
When I get the chance I'll test it out and report back, thanks.

**Comment by drobilla** [comment]
If that didn't fix it, 3e5cf34 surely did.  Please reopen or file a new ticket if you keep having crashes like this.
